extensions/stackdriver/stackdriver.cc

```diff
@@ -368,6 +368,18 @@ bool StackdriverRootContext::onConfigure(size_t size) {
   return true;
 }
 
+bool StackdriverRootContext::initializeLogFilter() {
+  uint32_t token = 0;
+  if (createExpression(config_.access_logging_filter_expression(), &token) !=
```

Comment:
Do we need to check if config_ has filter expression first?

Reply:
added check.